server/server.go

```diff
@@ -661,6 +673,19 @@ func (s *Server) stopRaftCluster() {
 	s.cluster.Stop()
 }
 
+// IsServiceEnabled returns whether the service is enabled.
+func (s *Server) IsServiceEnabled(serviceName string) bool {
+	if len(s.services) == 0 {
```

Comment:
I think it's a bit of a hack to start all services by default if the length is equal to 0. Perhaps it would be better to pass in all the services names that have been split for the legacy pd server?

Reply:
IMO, they are in different modes, in PD only mode, there is no need to introduce service conception.

Reply:
But I will reconsider your suggestion.

Reply:
Maybe we need to know the mode?